packages/path_provider/path_provider/README.md

```diff
@@ -34,10 +38,13 @@ Directories support by platform:
 | External Storage Directories | ✔️ | ❌ | ❌ | ❌️ | ❌️ |
 | Downloads | ❌ | ❌ | ✔️ | ✔️ | ✔️ |
 
+### Linux
```

Comment:
nit: Is this section needed? It is suppose to say more?

Reply:
Oops, thanks! I started to add a section saying it required the host OS to implement the XDG directory spec, then remembered it had a fallback so that wasn't actually true, but forgot that I'd half-written this when I changed the table entry. Removed.